BigQuery/tests/Unit/ExtractJobConfigurationTest.php

```diff
@@ -74,14 +74,15 @@ public function testFluentSetters()
         ];
         $this->expectedConfig['configuration']['extract'] = $extract
             + $this->expectedConfig['configuration']['extract'];
-        $this->config
+
+        $this->assertInstanceOf(ExtractJobConfiguration::class, $this->config
```

Comment:
Would we be able to make this assertion separate from the usage of the setters please?

Reply:
done!